extensions/amp-inputmask/validator-amp-inputmask.protoascii

```diff
@@ -46,6 +46,10 @@ tags: {
         "payment-card"  # A named mask followed by
         "\\s"           # a single whitespace
         "[^\\s]+)"      # followed by one or more non-whitespace characters.
+        "|([^\\s]+\\sdate-dd-mm-yyyy|date-dd-mm-yyyy\\s[^\\s]+)"
```

Comment:
I don't believe it's necessary to repeat the pre and post amble for whitespaces for each named mask. Instead these new ones could be put with `payment-card`. Please verify with tests of course.

E.g.
```
        "([^\\s]+"          # One or more non-whitespace characters before
        "\\s"               # a single whitespace
        "(?:"               # non capturing group of one of the following:
        "date-dd-mm-yyyy|"  # a named mask or
        "date-mm-dd-yyyy|"  # a named mask or
        "date-mm-yy|"       # a named mask or
        "date-yyyy-mm-dd|"  # a named mask or
        "payment-card"      # a named mask
        ")"                 # end of non capturing group.
        "|"                 # Or (to cover the other case):
        "(?:"               # non capturing group of one of the following:
        "date-dd-mm-yyyy|"  # a named mask or
        "date-mm-dd-yyyy|"  # a named mask or
        "date-mm-yy|"       # a named mask or
        "date-yyyy-mm-dd|"  # a named mask or
￼	    "payment-card"      # A named mask followed by
        ")"                 # end of non capturing group.
￼	    "\\s"               # a single whitespace
￼	    "[^\\s]+)"          # followed by one or more non-whitespace characters.
```

I also noticed this does not catch the attribute value of `payment-card  date-dd-mm-yy`. The `single whitespace` should become `\\s+  # one or more whitespaces, followed by`. Probably want to add a test case for this.

Reply:
Thanks for the suggestion, this should help prevent mistakes when adding future named masks. I'll make this change in a followup PR.